tests/payloads/bmcapi/servers/servers_post.json

```diff
@@ -63,6 +63,11 @@
           "installOsToRam": false,
           "cloudInit": {
             "userData": "data"
+          },
+          "esxi": {
```

Comment:
should this also be part of the responses, or just the request?
I think also cloudInit and maybe some others are missing from this JSON payload in responses

Reply:
ah, forgot to add it to the response for the servers_post - ill update it

Reply:
added